src/js/components/AppPageComponent.jsx

```diff
@@ -88,9 +88,10 @@ var AppPageComponent = React.createClass({
 
   scaleApp: function () {
     var model = this.props.model;
+    /*eslint-disable no-alert */
     var instancesString = prompt("Scale to how many instances?",
       model.get("instances"));
-
+    /*eslint-enable no-alert */
```

Comment:
I think I mentioned this before, may not in this PR.
But rather than merging in something that we know will change is is relatively small in effort, can we just create a Util for these things `prompt`, `alert`, `confirm`?
Otherwise we will have all these comments and then will have to remove them later in favour of what I'm suggesting.